src/main/scala/scala/tools/refactoring/util/Casts.scala

```diff
@@ -0,0 +1,9 @@
+package scala.tools.refactoring.util
+
```

Comment:
I like my version better too (mostly because writing `x.tryMatch { ... }` looks more natural to me then `condOpt(x) { ... }`), but then I also think that one should stick to standards, and not reinvent the wheel, so I dumped it, and replaced everything with `condOpt`. Thanks for pointing out!